locale.html

```diff
@@ -90,6 +90,55 @@ <h1><ins>TimeZonesOfLocale ( _loc_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-direction-of-locale" aoid=DirectionOfLocale>
+      <h1><ins>DirectionOfLocale ( _loc_ )</ins></h1>
+
+      <emu-alg>
+        1. <ins>If the default general ordering of characters (characterOrder) within a line in the _loc_ is right-to-left, then return *"rtl"*.</ins>
+        1. <ins>Return *"ltr"*.</ins>
+      </emu-alg>
+
+    </emu-clause>
+
+    <emu-clause id="sec-week-info-of-locale" aoid=WeekInfoOfLocale>
+      <h1><ins>WeekInfoOfLocale ( _loc_ )</ins></h1>
+
+      <emu-alg>
+        1. <ins>Return a record with fields base on _loc_ according to <emu-xref href="#table-locale-weekinfo-record"></emu-xref></ins>
+      </emu-alg>
+
+      <emu-table id="table-locale-weekinfo-record">
+        <emu-caption><ins>Record returned by WeekInfoOfLocale</ins></emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th><ins>Field</ins></th>
```

Comment:
```suggestion
              <th><ins>Field Name</ins></th>
```

Reply:
Also maybe add a separate "Value" column to describe the value type and rename the current "Value" column to "Meaning", cf. <https://tc39.es/ecma262/#sec-completion-record-specification-type> and other similar tables in ECMA-262.